gtfs/Governance/change-process.md

```diff
@@ -0,0 +1,379 @@
+    * The [Advocate](roles.md/#advocate) also announces the end of the vote in the [GTFS Changes mailing list](https://groups.google.com/g/gtfs-changes) discussion thread, including the result.
+
+5. **Failed Vote**  
+     
+    * If the vote fails, the [Advocate](roles.md/#advocate) can choose to:  
+        1. Continue working on the proposal, or  
+        2. Abandon the proposal.  
+    * The [Advocate](roles.md/#advocate) must announce their decision in the Pull Request comment section and in the [GTFS Changes mailing list](https://groups.google.com/g/gtfs-changes) discussion thread.
+
+**<ins>Requirements</ins>** 
+
+The vote must meet the following conditions:
+
+| Requirement | Details |
+| :---- | :---- |
+| **Consensus Principle** | The vote is based on **unanimous consensus**. |
+| **Vote Passing Criteria** | The vote passes when all contributors vote **\+1**. |
+| **Vote Failing Criteria** | The vote fails when any contributor votes **\-1**. |
+| **Voting Format** | Votes must be formatted as follows: <br>- *“+1 or \-1, Organization Name, Contributor Type (Consumer, Producer, or General Contributor), Link to Produced Feed or Consuming Application”* |
+| **Voting Against** | Contributors providing a negative vote (-1) must give actionable feedback. <br>- Actionable feedback is practical and constructive, providing concrete observations or suggestions to help solve the identified issue:  <br> - “*This proposal does not respect the backward compatibility principle of GTFS and we propose to create a separate file instead.*” |
+| **Minimum Votes** | At least 5 votes must be cast. |
+| **Participant Composition** | At least 2 consumers and 2 producers must participate in the vote. |
+| **Advocate's Vote** | The Advocate cannot vote for their own proposal. |
+| **Invalid Votes** | Votes are considered invalid if:  <br>- A contributor casts their vote outside (before or after) the official voting period. <br>- An individual or organization votes more than once (Only one vote per individual or organization is permitted.)  |
+| **Minimum Voting Period**  | The voting period must last at least **14 full calendar days** and end at **23:59:59 UTC**. |
+
+### Step 2.5: Testing
+
+![](assets/governance-process-step-2.5.svg)
+
+> Note: Not applicable to Track B: Non-Functional Changes and Track C: Documentation Maintenance
+
+One GTFS [Producer](roles.md/#producers) and one GTFS [Consumer](roles.md/#consumers) volunteer to serve as [First Adopters](roles.md/#first-adopter) by implementing the proposed changes for testing.
+
+**<ins>Actions</ins>** 
+
+1. **Tester Confirmation**  
+     
+    * The [Advocate](roles.md/#advocate) confirms the identity of the [First Adopters](roles.md/#first-adopter) which will test the changes in the Pull Request comment section.  
+    
+2. **Testing**
+
+    * The [First Adopters](roles.md/#first-adopter) apply and test the changes in a public-facing environment.  
```

Comment:
Should be specific about what "public-facing environment" is. Not confident on this language, but perhaps:

```suggestion
    * The [First Adopters](roles.md/#first-adopter) apply and test the changes in a public-facing environment. For the Producer, this means a public GTFS feed; for the Consumer, this means a public-facing, production version of an application.
```